how-to/how-to-use-kata-containers-with-nemu.md

```diff
@@ -0,0 +1,59 @@
+# Kata Containers with NEMU
+Kata Containers relies by default on Qemu hypervisor in order to spawn the virtual machines running containers. [NEMU](https://github.com/intel/nemu) is a fork of Qemu focused on increasing the security through 3 different ways:
+- Reducing the number of lines of code.
+- Removing all legacy devices.
+- Reducing the emulation as far as possible.
+
+As the goal for Kata Containers is to increase the security, this is a logical step to get Kata relying on NEMU as its hypervisor.
```

Comment:
Again, "increase security" (or maybe "increase the security of running containers") sounds better to me as it's a general goal.

Reply:
@sboeuf I would say reduce the trusted computing base of the hypervisor by .... instead of saying increasing security.

Reply:
I removed it completely.